backend/data/src/main/kotlin/io/tolgee/security/constants.kt

```diff
@@ -0,0 +1,4 @@
+package io.tolgee.security
+
+const val PROJECT_API_KEY_PREFIX = "pak_"
+const val PAT_PREFIX = "pat_"
```

Comment:
If we start prefixing all the tokens emitted by Tolgee, it might be a good idea to add a shared identifiable prefix, so tools like GitHub's Code Scanning can be configured to easily pick up leaked tokens (e.g.: `tgpak_`, `tgpat_`)

Reference: [GitHub blog post on new GitHub Token Format](https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/#identifiable-prefixes)

Reply:
Good point!